### PR TITLE
Redesign backup sheet on Android

### PR DESCRIPTION
**Pull request by osdnk**
I think it will be the easiest way not to expand the sheet but open a new full-screen form instead (that's how we do it in setting)

demo:https://streamable.com/4pjung

### CODE REVIEW

File: src/screens/BackupSheet.js [hunk 1]
```diff
@@ -109,10 +131,18 @@ export default function BackupSheet() {
...
-    () => setParams({ step: WalletBackupStepTypes.manual }),
-    [setParams]
-  );
+  const onManualBackup = useCallback(() => {
+    if (android) {
+      goBack();
+      navigate(Routes.BACKUP_SCREEN, {
+        nativeScreen: true,
+        step: WalletBackupStepTypes.manual,
+        walletId,
```
Comment (brunobar79) [inline src/screens/BackupSheet.js:140]: and here

File: src/screens/BackupSheet.js [hunk 2]
```diff
@@ -67,17 +70,36 @@ export default function BackupSheet() {
...
+    }
+  }, [goBack, navigate, setParams, walletId]);
 
   const handlePasswordNotFound = useCallback(() => {
-    setParams({
-      missingPassword: true,
-      step: WalletBackupStepTypes.cloud,
-    });
-  }, [setParams]);
+    console.log('YY');
```
Comment (brunobar79) [inline src/screens/BackupSheet.js:87]: 😮

File: src/screens/BackupSheet.js [hunk 3]
```diff
@@ -67,17 +70,36 @@ export default function BackupSheet() {
...
+    if (android) {
+      goBack();
+      navigate(Routes.BACKUP_SCREEN, {
+        missingPassword: true,
+        nativeScreen: true,
+        step: WalletBackupStepTypes.cloud,
+        walletId,
```
Comment (brunobar79) [inline src/screens/BackupSheet.js:94]: and here

File: src/screens/BackupSheet.js [hunk 4]
```diff
@@ -67,17 +70,36 @@ export default function BackupSheet() {
...
-    () => setParams({ step: WalletBackupStepTypes.cloud }),
-    [setParams]
-  );
+  const handleNoLatestBackup = useCallback(() => {
+    if (android) {
+      goBack();
+      navigate(Routes.BACKUP_SCREEN, {
+        nativeScreen: true,
+        step: WalletBackupStepTypes.cloud,
+        walletId,
```
Comment (brunobar79) [inline src/screens/BackupSheet.js:79]: Previously we were not passing the walletId param here.

**Review by brunobar79 — APPROVED** [review]
VIdeo looks great and code looks good. Added a comment about a param that we were not passing previously. 

Skylar will take a look too

**Comment by skylarbarrera** [comment]
Ran into this breaking issue when opening through settings -> backup on Samsung. Like what you're doing here a lot though. Lmk when its fixed and ill check again + iOS

https://cloud.skylarbarrera.com/Screen_Recording_20201013-144003.mp4

**Pull request merged**